lib/build_manager.py

```diff
@@ -44,8 +45,11 @@ def _build_packages(self, distro, packages):
 
         # create package builder based on distro
         if distro.name == "CentOS":
-            mock_config_file = CONF.get('build_packages').get(
-                'mock_config').get(distro.name).get(distro.version)
+            mock_config_file = os.path.join(
+                "mock_configs", distro.name, distro.version,
+                "%s-%s-%s.cfg" % (distro.name, distro.version, distro.architecture))
+            if not os.path.isfile(mock_config_file):
+                raise exception.BaseException("Mock config file not found at %s" % mock_config_file)
             package_builder = mockbuilder.Mock(mock_config_file)
```

Comment:
This is pinning the configuration file path to a value user cannot change, making it difficult for user to build using different configuration files.

👎